conf/gulp-tasks/bundle/minifytaskfactory.js

```diff
@@ -0,0 +1,26 @@
+const getBundleName = require('../utils/bundlename');
+const { minifyBundle } = require('./bundle');
+
+/**
+ * A factory class that provides Gulp tasks to minify the SDK bundles.
+ */
+class MinifyTaskFactory {
+  constructor (locale) {
+    this._locale = locale;
+  }
+
+  /**
+   * Provides a Gulp task to minify an SDK bundle of the specified type.
+   *
+   * @param {BundleType} bundleType The type of SDK bundle to minify.
+   * @returns {Function} Gulp task for minifying the requested SDK bundle.
+   */
+  minify (bundleType) {
```

Comment:
I'd say lets move minifyBundle into here if possible. I think that fits with this class's responsibility. Additionally, it allows us to clarify the role of bundle.js: it only provides Gulp tasks for bundling.

Reply:
Totally, moved it!